### ISSUE TITLE
read.flowSet Vague Error Message If Features Differ

### ISSUE THREAD

**Issue by mikejiang** [RGLab/flowCore#100]
On 02/20/2018 11:00 PM, Dario Strbenac wrote:
> Good day,
>
> read.flowSet is used to import the FCS files, then a vague and unhelpful error message is reported. The cause of this is a differing number of features recorded for samples in the training and test sets. Could the input checking of read.flowSet be improved to allow such a problem to be better communicated to the user by printing a more meaningful error message?
>
>> files
> [1] "Data23_Panel1_base_NR1_Patient6.fcs" 
> [2] "Data29_Panel1_base_NR6_Patient16.fcs"
>
>> tables[[1]]
> flowFrame object 'Data23_Panel1_base_NR1_Patient6.fcs'
> with 5015 cells and 36 observables:
>> tables[[2]] # The samples belonging to the test (Data29) dataset have one less feature than samples in the training dataset (Data23), although the panel ID is the same.
> flowFrame object 'Data29_Panel1_base_NR6_Patient16.fcs'
> with 5585 cells and 35 observables:
>
>> dataset <- read.flowSet(files)
> Error in dim(colNames) <- c(ncol(from[[frameList[[1]]]]), length(frameList)) : 
>   dims [product 72] do not match the length of object [2]
>
> --------------------------------------
> Dario Strbenac
> University of Sydney
> Camperdown NSW 2050
> Australia
